tests/func/test_external_repo.py

```diff
@@ -128,60 +128,56 @@ def test_relative_remote(erepo_dir, tmp_dir):
             assert fd.read() == "contents"
 
 
-def test_shallow_clone_branch(erepo_dir):
+def test_shallow_clone_branch(erepo_dir, mocker):
     with erepo_dir.chdir():
         with erepo_dir.branch("branch", new=True):
             erepo_dir.dvc_gen("file", "branch", commit="create file on branch")
         erepo_dir.dvc_gen("file", "master", commit="create file on master")
 
     url = os.fspath(erepo_dir)
+    mock_clone = mocker.patch.object(Git, "clone", wraps=Git.clone)
```

Comment:
See [`mocker.spy`](https://github.com/pytest-dev/pytest-mock/#spy) instead  of `wraps=`.